tests/components/vacuum/test_xiaomi_miio.py

```diff
@@ -330,3 +331,13 @@ def test_xiaomi_specific_services(hass, caplog, mock_mirobo_is_on):
         [mock.call.Vacuum().manual_control_once(control_once)], any_order=True)
     mock_mirobo_is_on.assert_has_calls(status_calls, any_order=True)
     mock_mirobo_is_on.reset_mock()
+
+    control = {"zone": [[123, 123, 123, 123]], "repeats": 2}
```

Comment:
The service payload here is completely different from the one shown above...

Reply:
why?
In test, I have set zone and repeats.. I have omitted only entity_id